backends/ebpf/targets/xdp_target.py

```diff
@@ -0,0 +1 @@
+/home/mbudiu/git/p4c/extensions/p4c-xdp/xdp_target.py
```

Comment:
This file appears to have been added unintentionally

Reply:
yes, I keep adding it.